src/sidebar/components/sidebar-content.js

```diff
@@ -234,27 +110,33 @@ function SidebarContentController(
   // Re-fetch annotations when focused group, logged-in user or connected frames
   // change.
   $scope.$watch(
-    () => [groups.focused(), store.profile().userid, ...store.searchUris()],
-    ([currentGroup], [prevGroup]) => {
-      if (!currentGroup) {
-        // When switching accounts, groups are cleared and so the focused group
+    () => [
+      store.getState().focusedGroupId,
```

Comment:
There is a `focusedGroupId` selector in the store that you can use instead of accessing the state directly. It currently just returns the relevant state value.